### PR TITLE
HP方針項目 文章の修正

### PR DESCRIPTION
**Pull request by Omuretsu**
表題のとおりです。

### CODE REVIEW

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/bans)
- [server/policies/index.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/)
- [server/policies/permissions.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/permissions)
- [server/policies/privacy.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/privacy)

File: server/policies/index.md [hunk 1]
```diff
@@ -64,8 +64,9 @@ jao Minecraft Server（以下、「当サーバ」と呼びます）は、これ
 サービス提供上の免責事項です。
 
 - 運営は利用者が本サービスを利用したことに起因して、利用者に生じたあらゆる損害や障害について一切の責任を負いません。
-- 運営は本サービスの内容について、瑕疵(かし)もしくはバグがないことを保証しておりません。それらの瑕疵やバグを出来る限り除去する努力義務を負いますが、それらによって生じた一切の問題についての責任を負いません。
+- 運営は本サービスの内容について、瑕疵(かし)もしくはバグがないことを保証しておりません。それらの瑕疵やバグをできる限り除去する努力義務を負いますが、それらによって生じた一切の問題についての責任を負いません。
 - 運営は、運営の判断によって利用者の行動を制限でき、運営の判断によって制限されていた特定のサービスを利用できるようにすることができるものとします。また、運営は不審な動作が見受けられる利用者の行動を通知なく規制する場合があります。
-- 運営は、現実世界での活動をある程度優先し、それに支障がでない形でサービスの提供を行うため、対応を行えないタイミングや時間帯というものがあります。利用者は運営に対して様々な問い合わせ等を行う場面があるかと思いますが、その際に短時間に複数回の問い合わせ、暴言等が見られた場合は必要に応じて発言権の取り上げ、サービスの利用を制限します。これは必要以上に問い合わせ等が行われた場合、本来不要な運営作業が発生するためです。
+- 運営は、現実世界での活動をある程度優先し、それに支障がでない形でサービスの提供を行うため、対応を行えないタイミングや時間帯というものがあります。  
+利用者は運営に対して、さまざまな問い合わせ等を行う場面があるかと思います。その際、短時間に複数回の問い合わせや暴言等が見られた場合は、必要に応じて発言権の取り上げやサービスの利用を制限します。これは必要以上に問い合わせ等が行われた場合、本来不要な運営作業が発生するためです。
```
Comment (book000) [inline server/policies/index.md:70]: ここ、改行していて表示上でリスト表示が切れてますけど意図したものですか？
https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/#免責事項

**Review by book000 — CHANGES_REQUESTED** [review]

File: server/policies/index.md (hunk 1 above)
Comment (Omuretsu) [reply server/policies/index.md]: Fixed🙇‍♂️

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/bans)
- [server/policies/index.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/)
- [server/policies/permissions.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/permissions)
- [server/policies/privacy.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/privacy)

**Comment by jaotan** [comment]
# 変更のあったファイルプレビュー
- [server/policies/bans.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/bans)
- [server/policies/index.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/)
- [server/policies/permissions.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/permissions)
- [server/policies/privacy.md](https://dev.jaoafa.com/jaoweb/jaoafa/pulls/85/server/policies/privacy)

**Review by book000 — APPROVED** [review]

**Pull request merged**